doc/man3/EVP_PKEY_size.pod

```diff
@@ -2,7 +2,8 @@
 
 =head1 NAME
 
-EVP_PKEY_size, EVP_PKEY_bits, EVP_PKEY_security_bits
+EVP_PKEY_size, EVP_PKEY_bits, EVP_PKEY_security_bits,
+EVP_PKEY_get_params, EVP_PKEY_gettable_params
```

Comment:
Why in this file rather than adding to `EVP_PKEY_gettable_params.pod`?

I am not opposed to unify them in a single pod file, but I am not sure I see the logical connection between `EVP_PKEY_size` and `EVP_PKEY_get_params`.

If we unify, EVP_PKEY_gettable_params.pod` should be removed.

Reply:
`EVP_PKEY_gettable_params()` was developed separately by both @slontis and me, and our approaches regarding the manual were different.  I haven't dealt with it, 'cause I'm ambivalent which way to go.

Reply:
Fixed